utils/context.py

```diff
@@ -29,8 +29,8 @@ def search_context(context, content=None, timestamp=None):
 
         return idx
 
-    for c in context:
-        c["timestamp"] = str(c["timestamp"])
+    # for c in context:
+    #     c["timestamp"] = str(c["timestamp"])
```

Comment:
I don't remember why this was here, probably for readability. Did this change slip in by error?